Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -493,6 +501,11 @@ open class ChatMessageContentView: _View, ThemeProvider {
         textView?.font = textFont
         textView?.text = content?.textContent
         
+        if markdownFormatterEnabled, markdownFormatter.containsMarkdown(text: content?.textContent ?? "") {
+            let markdownText = markdownFormatter.format(from: content?.textContent ?? "")
+            textView?.attributedText = markdownText
+        }
```

Comment:
Let's assign either `attributedText` or `text`. Currently, we always assign `text` the live above and then conditionally override it.

Reply:
Or maybe always use `attributedText` but with a font if markdown is not detected?

Reply:
I decided to do it that way because whenever you set `attributedString`  and if you have values in `text` , `textColor`  or `textFont` , they’ll get overridden as mentioned on Apple’s docs. But I see your point and so I will wrap this with an if statement; if there is markdown, set the `attributedString`  else set text normally as we currently have.
The reason I’m not setting attributedText  on all cases even if there isn’t markdown syntax in the `String` is because I would have to wrap the String around a `NSMutableAttributedString`  first, add the existing textFont  and textColor  properties as additional attributes and then assign. Feels more verbose IMHO and there is no problem if either text  or attributedText  are set, but only one of them.